windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -143,10 +150,14 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 # unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
-& $GEM_BAT update --system 2.4.8
-& $GEM_BAT install fluentd:0.14.1 --no-ri --no-rdoc --no-document
+# Install a specific version of serverengine with fluentd as there is a compatibility issue with later versions
+# See: https://github.com/fluent/fluentd/issues/1195
+& $GEM_BAT install serverengine:1.6.4 fluentd:0.14.1 --no-ri --no-rdoc --no-document
```

Comment:
I can't help but wonder whether this can just be driven off the `Gemfile` and whether we can get the gem command to just install all of the dependencies... Something along the lines of http://guides.rubygems.org/command-reference/#gem-dependency-files?